packages/react/src/ButtonGroup/index.ts

```diff
@@ -0,0 +1 @@
+export * from './ButtonGroup'
```

Comment:
Can you also add the same export statement in `src/index.ts` please?

It won't get published in the package without that.

Also can't see a changeset here, could you add one in too please? [Example here from ComparisonTable](https://github.com/primer/brand/blob/6e7bdecee69458c1432d34f598479ac44029e20f/.changeset/honest-weeks-shake.md).

Reply:
done